### DISCUSSION TITLE
Fluid Flow Simulation Using Physics-Informed Neural Networks (Project 252)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#117] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 252](https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/blob/main/projects/Fluid%20Flow%20Simulation%20Using%20Physics-Informed%20Neural%20Networks)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by leo10manyam** [comment]
Hey everyone!

Thanks for checking out the MATLAB and Simulink Challenge projects! I’m K Subramanyam, a student from Bapuji Institute of Engineering and Technology, Bangalore, and I’m excited to start working on Fluid Flow Simulation Using Physics-Informed Neural Networks.

I’d love to connect and collaborate with anyone who’s interested in this topic or has insights to share. Let’s learn from each other and build something amazing together!

**Reply by mohamed-khoualed** [reply]
yeah me too

**Comment by arturofburgos** [comment]
Hi, I am a student at University of Illinois and I am interested in working and collaborating on this project!

**Reply by mohamed-khoualed** [reply]
Please  help 
have we to take the hole datasets or chosse one of them?
[https://www.kaggle.com/datasets/ryleymcconkey/ml-turbulence-dataset/versions/3/data](url)

**Comment by mishaq151** [comment]
I am also interested in this work.

**Comment by mohamed-khoualed** [comment]
I'm working on this project

**Comment by Rivu-95** [comment]
Hi, I am a student at Birla Institute of Technology, Mesra and I am interested in working and collaborating on this project!

**Reply by mohamed-khoualed** [reply]
yeah let's do it together

**Comment by Elan071234** [comment]
hey guys,I am having a small query...since they just mentioned using a dataset to feed in our PINN,can we just generate our own dataset using a simuation or we need to be relaible on datasets from Kaggle,John hopkins as mentioned in the problem statement

**Reply by robertogl** [reply]
Hi @Elan071234 ,

You definitely can, but Computational Fluid Dynamics (CFD) simulation it is not an easy task and you will need third party tools. If you feel comfortable, you are more than welcome to make your own simulation!

**Comment by minowau** [comment]
I want to work on this prject but would be willing more to join other's team who are dng this, If you want help or have space please count me in

**Comment by rowbarb** [comment]
Hi everyone. I'm a student at Stevens Institute of Technology and my group and I are also working on this project.
